editor.planx.uk/src/@planx/components/Pay/Editor.tsx

```diff
@@ -1,225 +1,392 @@
+      validateOnChange={false}
+      validateOnBlur={false}
+    >
+      {({ values, handleChange, setFieldValue, errors }) => (
+        <Form id="modal" name="modal">
+          <ModalSection>
+            <ModalSectionContent title="Payment" Icon={ICONS[TYPES.Pay]}>
+              <InputRow>
+                <Input
+                  format="large"
+                  placeholder="Page title"
+                  name="title"
+                  value={values.title}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <Input
+                  format="bold"
+                  placeholder="Banner title"
+                  name="bannerTitle"
+                  value={values.bannerTitle}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <RichTextInput
+                  placeholder="Banner description"
+                  name="description"
+                  value={values.description}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <Input
+                  format="data"
+                  name="fn"
+                  value={values.fn}
+                  placeholder="Data field for calculated amount"
+                  onChange={handleChange}
+                />
+              </InputRow>
+            </ModalSectionContent>
+            <ModalSectionContent>
+              <InputRow>
+                <Input
+                  format="large"
+                  placeholder="Instructions title"
+                  name="instructionsTitle"
+                  value={values.instructionsTitle}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <RichTextInput
+                  placeholder="Instructions description"
+                  name="instructionsDescription"
+                  value={values.instructionsDescription}
+                  onChange={handleChange}
+                />
+              </InputRow>
+            </ModalSectionContent>
+            <OptionButton
+              selected={values.hidePay}
+              onClick={() => {
+                setFieldValue("hidePay", !values.hidePay);
+              }}
+              style={{ width: "100%" }}
+            >
+              Hide the pay buttons and show fee for information only
+            </OptionButton>
+          </ModalSection>
+          <ModalSection>
+            <ModalSectionContent
+              title="GOV.UK Pay Metadata"
+              Icon={DataObjectIcon}
+            >
+              <Typography variant="subtitle2" sx={{ mb: 2 }}>
+                Include metadata alongside payments, such as VAT codes, cost
+                centers, or ledger codes. See{" "}
+                <Link href="#TODO">our guide</Link> for more details.
```

Comment:
Intentional `TODO` - will write something up in Notion, likely after initial PO testing round.

Reply:
I wonder if we can just link directly to Gov Pay docs here and save ourselves a custom doc? Eg https://docs.payments.service.gov.uk/reporting/#add-more-information-to-a-payment-39-custom-metadata-39-or-39-reporting-columns-39

A bit more technical, but probably ideal for councils to be used to consulting Gov Pay directly anyways?

Reply:
This is a really good shout tbh! Updated here [45a289b](https://github.com/theopensystemslab/planx-new/pull/2878/commits/45a289bb834a278f63e5038f162babb82e6987f5)

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/839dc8e0-3cff-4562-bf56-187b07d87b27)